### PR TITLE
Models group1

### PR DESCRIPTION
**Pull request by ryan-schlientz**

### CODE REVIEW

File: BattleSim/src/main/java/com/revature/models/Item.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	@Id
+	@Column(name = "i_id", updatable = false)
+	@SequenceGenerator(name = "item_seq", sequenceName = "item_seq")
+	@GeneratedValue(generator = "item_seq", strategy = GenerationType.SEQUENCE)
+	private int id;
+	private String name;
+	@Column(name = "ITEM_TYPE")
+	private String itemtype;
+	@Column(name = "STATS_ID")
+	private int statsid;
```
Comment (ryan-schlientz) [inline BattleSim/src/main/java/com/revature/models/Item.java:22]: I dont like this!

File: BattleSim/src/main/java/com/revature/repositories/ItemRepository.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import java.util.List;
+
+import com.revature.models.Item;
+
+public interface ItemRepository {
+	public int addItem(Item item);
+	public List<Item> getAllItems();
+	public Item getStats(int id);
+	public boolean updateStats(Item item);
+}
```
Comment (plboronski) [inline BattleSim/src/main/java/com/revature/repositories/ItemRepository.java:13]: Missing a delete method

**Review by mitch-greene92 — APPROVED** [review]

**Review by plboronski — CHANGES_REQUESTED** [review]

**Review by KMalazarte — APPROVED** [review]

File: BattleSim/src/main/java/com/revature/repositories/ItemRepository.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.revature.repositories;
+
+import java.util.List;
+
+import com.revature.models.Item;
+
+public interface ItemRepository {
+	public int addItem(Item item);
+	public List<Item> getAllItems();
+	public Item getStats(int id);
```
Comment (ryan-schlientz) [inline BattleSim/src/main/java/com/revature/repositories/ItemRepository.java]: Remove getStats. Add deleteItem

File: BattleSim/src/main/java/com/revature/repositories/ItemRepositoryImpl.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+			e.printStackTrace();
+		} finally {
+			sess.close();
+		}
+		
+		return s;
+	}
+
+	@Override
+	public boolean updateStats(Item item) {
```
Comment (chin3) [inline BattleSim/src/main/java/com/revature/repositories/ItemRepositoryImpl.java]: This should also be changed to item

**Review by devdavq — APPROVED** [review]

**Review by andrewvandyke27 — COMMENTED** [review]
do what ryan said

**Pull request merged**